src/PerformanceTests/Tests/Categories/DistributorFixture.cs

```diff
@@ -6,7 +6,6 @@ namespace Categories
     using Variables;
 
     [TestFixture(Description = "Distributor distribution", Category = "Performance")]
-    [Explicit]
```

Comment:
Please add this again, this is purely for testing purposes.

Reply:
ok, will readd.

Reply:
although I'd rather question the point of these tests then tbh.